src/tests/JIT/opt/OSR/invalidpromotion.csproj

```diff
@@ -6,25 +6,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="COMPlus_TieredCompilation" Value="1" />
+    <CLRTestEnvironmentVariable Include="COMPlus_TC_QuickJitForLoops" Value="1" />
+    <CLRTestEnvironmentVariable Include="COMPlus_TC_OnStackReplacement" Value="1" />
+    <CLRTestEnvironmentVariable Include="COMPlus_OSR_HitLimit" Value="1" />
+    <CLRTestEnvironmentVariable Include="COMPlus_TC_OnStackReplacement_InitialCounter" Value="1" />
   </ItemGroup>
-  <PropertyGroup>
-    <CLRTestBatchPreCommands><![CDATA[
-$(CLRTestBatchPreCommands)
-set COMPlus_TieredCompilation=1
-set COMPlus_TC_QuickJitForLoops=1
-set COMPlus_TC_OnStackReplacement=1
-set COMPlus_OSR_HitLimit=1
-set COMPlus_TC_OnStackReplacement=1
-set COMPlus_TC_OnStackReplacement_InitialCounter=1
-]]></CLRTestBatchPreCommands>
-    <BashCLRTestPreCommands><![CDATA[
-$(BashCLRTestPreCommands)
-export COMPlus_TieredCompilation=1
-export COMPlus_TC_QuickJitForLoops=1
-export COMPlus_TC_OnStackReplacement=1
-export COMPlus_OSR_HitLimit=1
-export COMPlus_TC_OnStackReplacement=1
-export COMPlus_TC_OnStackReplacement_InitialCounter=1
```

Comment:
This diff condenses 6 variables to 5 because `COMPlus_TC_OnStackReplacement` is defined twice.